lib/companion_web/jobs/process_opt_out.ex

```diff
@@ -42,13 +42,14 @@ defmodule Companion.Jobs.ProcessOptOut do
     msisdn <> "^^^ZAF^TEL"
   end
 
-  defp openhim_optout_from_contact(contact) do
+  defp openhim_optout_from_contact(contact, id) do
     %{
       "urns" => [nurse_urn | _],
       "fields" => %{
         "opt_out_date" => opt_out_date,
         "registered_by" => device_msisdn,
-        "facility_code" => facility_code
+        "facility_code" => facility_code,
+        "uuid" => uuid
```

Comment:
This need to be a level higher (the same level as `urns` and `fields`), as it is not a field, rather a direct property of the contact.